src/assets/js/presence.js

```diff
@@ -46,7 +45,7 @@ export default class Presence {
   }
 
   addReceivedUsers(data) {
-    self = this;
```

Comment:
We might need to all start doing 'module' global stuff instead of self... maybe we can do 
```javascript
var __this;

export default class Test {
    constructor (input) {
         __this = this;
}}
//?
```